### PR TITLE
JAVA-18113 Update spring-boot-swagger-keycloak module

### PR DESCRIPTION
**Pull request by anuragkumawat**

### CODE REVIEW

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/GlobalSecurityConfig.java [hunk 1]
```diff
@@ -1,59 +1,48 @@
...
-    /*
-     * re-configure Spring Security to use
-     * registers the KeycloakAuthenticationProvider with the authentication manager
-     */
-    @Autowired
-    void configureGlobal(AuthenticationManagerBuilder auth) {
-        KeycloakAuthenticationProvider provider = keycloakAuthenticationProvider();
-        provider.setGrantedAuthoritiesMapper(authoritiesMapper());
-        auth.authenticationProvider(provider);
+    @Bean
```
Comment (lor6) [inline spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/GlobalSecurityConfig.java:42]: is defining this bean necessary?

Comment (anuragkumawat) [reply spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/GlobalSecurityConfig.java]: redundant code has been removed

File: spring-boot-modules/spring-boot-swagger-keycloak/pom.xml
Comment (anastasiosioannidis) [inline spring-boot-modules/spring-boot-swagger-keycloak/pom.xml:1]: Could you fix the identation here?

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/OpenAPISecurityConfig.java [hunk 2]
```diff
@@ -1,22 +1,16 @@
 package com.baeldung.swaggerkeycloak;
-
```
Comment (anastasiosioannidis) [inline spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/OpenAPISecurityConfig.java:2]: add this separating line back

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml [hunk 3]
```diff
@@ -1,8 +1,13 @@
+server:
+   port: 8081
 keycloak:
```
Comment (anastasiosioannidis) [inline spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml:3]: add a blank line above keycloak stuff

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
+  auth-server-url: http://localhost:8080 # Keycloak server url
+  realm: SpringBootKeycloak # Keycloak Realm
+    
+spring:
+   security:
+      oauth2:
+         resourceserver:
+            jwt.issuer-uri: http://localhost:8080/realms/SpringBootKeycloak
+
+
```
Comment (anastasiosioannidis) [inline spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml:13]: add

springdoc:
  swagger-ui:
    oauth:
      client-id: login-app

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/OpenAPISecurityConfig.java (hunk 2 above)
Comment (anuragkumawat) [reply spring-boot-modules/spring-boot-swagger-keycloak/src/main/java/com/baeldung/swaggerkeycloak/OpenAPISecurityConfig.java:2]: done

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml (hunk 3 above)
Comment (anuragkumawat) [reply spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml:3]: done

File: spring-boot-modules/spring-boot-swagger-keycloak/pom.xml
Comment (anuragkumawat) [reply spring-boot-modules/spring-boot-swagger-keycloak/pom.xml:1]: I am using https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/eclipse 
formatter , let me know if I should use some other formatter

File: spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml (hunk 4 above)
Comment (anuragkumawat) [reply spring-boot-modules/spring-boot-swagger-keycloak/src/main/resources/application.yml:19]: done

**Pull request merged**